src/components/Garden/Feed/ProposalCardModalSupporters.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { GU, Modal, textStyle, useViewport } from '@1hive/1hive-ui'
+import { ProposalType } from '@/hooks/constants'
+import { useGardenState } from '@/providers/GardenState'
+import IdentityBadge from '@/components/IdentityBadge'
+import { formatTokenAmount } from '@utils/token-utils'
+
+type ModalSupportersProps = {
+  proposal: ProposalType
+  visible: boolean
+  onClose: () => void
+}
+
+const ModalSupporters = ({
+  proposal,
+  visible,
+  onClose,
+}: ModalSupportersProps) => {
+  const { width } = useViewport()
+  const { config } = useGardenState()
+  const { requestToken } = config.conviction
+
+  return (
+    <Modal
+      padding={5 * GU}
+      visible={visible}
+      width={Math.min(55 * GU, width - 40)}
+      onClose={onClose}
+    >
+      <div>
+        <h3
+          css={`
+            ${textStyle('title2')}
+            margin-top: 24px;
+            margin-bottom: 8px;
+          `}
+        >
+          Supporters
+        </h3>
+
+        <div
+          style={{
+            overflowY: 'scroll',
+            height: '300px',
+            paddingRight: '10px',
+          }}
+        >
+          {proposal?.stakes.map((stake, i) => (
```

Comment:
What do you think of sorting the stakes in descending order? And I wonder if we should filter stakes smaller than a certain threshold, something around `0,01` maybe?

Reply:
Ok, yes we could do that of course. I will make that change.

Reply:
Just made those changes @0xGabi